docs/additional-functionality/rapids-udfs.md

```diff
@@ -174,31 +173,51 @@ To enable _GPU Scheduling for Pandas UDF_, you need to configure your spark job
     --py-files ${SPARK_RAPIDS_PLUGIN_JAR}
     ```
 
-3. Enable GPU Scheduling for Pandas UDF.
+3. Enable GPU Assignment(Scheduling) for Pandas UDF.
 
     ```shell
     ...
     --conf spark.rapids.python.gpu.enabled=true \
-    --conf spark.rapids.python.memory.gpu.pooling.enabled=false \
-    --conf spark.rapids.sql.exec.ArrowEvalPythonExec=true \
-    --conf spark.rapids.sql.exec.MapInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.AggregateInPandasExec=true \
-    --conf spark.rapids.sql.exec.FlatMapCoGroupsInPandasExec=true \
-    --conf spark.rapids.sql.exec.WindowInPandasExec=true
     ```
 
-Please note the data transfer acceleration only supports scalar UDF and Scalar iterator UDF currently. 
-You could choose the exec you need to enable.
+Please note the data transfer acceleration only supports `scalar UDF` and `Scalar iterator UDF` currently. 
```

Comment:
We could have a table to show the status of the data transfer acceleration for all the 6 plans, instead of this notice. You can find the latest status from the commments of issue https://github.com/NVIDIA/spark-rapids/issues/305 .

Reply:
Good suggestion, made a status table for them.